man/udev.xml

```diff
@@ -465,6 +465,10 @@
             that is enforced on <filename>systemd-udevd.service</filename>.</para>
             <para>Please also note that <literal>:=</literal> and <literal>=</literal> are clearing
             both, program and builtin commands.</para>
+            <para>In order to activate long-running processes from udev rules, provide a service unit, and
+            pull it in from a udev device using the <varname>SYSTEMD_WANTS</varname> device property. See
+            <citerefentry><refentrytitle>systemd.device</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+            for details.</para>
```

Comment:
Hmm, for a clean service you'd have to set tons of properties (i.e. starting with Description=, but also all deps, since udev is early boot and these services would probably be too). Cramping that all into one udev property sounds less than ideal, and a unit file on disk the better approach, I think.

Reply:
Description could be set automatically to something like "Udev handler for $DEVICE"... But yeah, this would require some thought to be useful.